kqueen/engines/eks.py

```diff
@@ -0,0 +1,220 @@
+from kqueen.config import current_config
+from kqueen.engines.base import BaseEngine
+import boto3
+
+import logging
+import yaml
+
+logger = logging.getLogger('kqueen_api')
+config = current_config()
+
+STATE_MAP = {
+    'CREATING': config.get('CLUSTER_PROVISIONING_STATE'),
+    'ACTIVE': config.get('CLUSTER_OK_STATE'),
+    'DELETING': config.get('CLUSTER_DEPROVISIONING_STATE'),
+    'FAILED': config.get('CLUSTER_ERROR_STATE'),
+    'UPDATED': config.get('CLUSTER_UPDATING_STATE')
```

Comment:
`'UPDATING'` ?

Reply:
for what i know the engine requires an updating state, so either we leave it or we need to put it to unknown